index.d.ts

```diff
@@ -8,6 +9,7 @@ interface RoundCheckboxProps {
     iconColor?: string,
     borderColor?: string,
     checked?: boolean,
+    style?: StyleProp<ViewStyle>;
```

Comment:
this should also be mentioned in readme, thanks

Reply:
I was thinking about adding that but had forgotten before that PR. I'll update it